llvm/test/CodeGen/DirectX/countbits.ll

```diff
@@ -0,0 +1,31 @@
+; RUN: opt -S -dxil-op-lower -mtriple=dxil-pc-shadermodel6.3-library %s | FileCheck %s
+
+; Make sure dxil operation function calls for countbits are generated for all integer types.
+
+; Function Attrs: nounwind
+define noundef i16 @test_countbits_short(i16 noundef %a) {
+entry:
+; CHECK:call i16 @dx.op.unary.i16(i32 31, i16 %{{.*}})
+  %elt.ctpop = call i16 @llvm.ctpop.i16(i16 %a)
+  ret i16 %elt.ctpop
+}
+
+; Function Attrs: nounwind
```

Comment:
It doesn't look like this test uses the `Function Attrs`.  If we don't then please remove them.